orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -266,7 +266,16 @@ class OperationsController {
 
   private void decorateBuildInfo(Map trigger) {
     if (trigger.master && trigger.job && trigger.buildNumber) {
-      def buildInfo = buildService.getBuild(trigger.buildNumber, trigger.master, trigger.job)
+      def buildInfo
+      try {
+        buildInfo = buildService.getBuild(trigger.buildNumber, trigger.master, trigger.job)
+      } catch (RetrofitError e) {
+        if (e.response?.status == 404) {
+          throw new IllegalStateException("Build " + trigger.buildNumber + " for " + trigger.master + "/" + trigger.job + " not found")
```

Comment:
why not use string interpolation?

Reply:
Because groovy g strings sometimes behave weird and I don't like them so I often just use the + like a noob.

Reply:
👍 just curious - since I see diff patterns in the code base